scripts/variantstore/wdl/GvsQuickstartHailIntegration.wdl

```diff
@@ -30,12 +30,22 @@ workflow GvsQuickstartHailIntegration {
             call_set_identifier = branch_name
     }
 
-    call CreateAndTieOutVds {
+    call CreateVds {
         input:
             branch_name = branch_name,
             use_classic_VQSR = use_classic_VQSR,
             avro_prefix = GvsExtractAvroFilesForHail.avro_prefix,
             vds_destination_path = GvsExtractAvroFilesForHail.vds_output_path,
+    }
+
+    call ValidateVds {
+        input:
+            vds_path = GvsExtractAvroFilesForHail.vds_output_path,
+    }
+
+    call TieOutVds {
+        input:
+            vds_path = GvsExtractAvroFilesForHail.vds_output_path, ## TODO why is this coming out of the avro task?
```

Comment:
This is coming out of the `GvsExtractAvroFilesForHail.wdl` *workflow*. That workflow was originally created to do what its name suggests but over time had its responsibilities expanded to also generate some Hail scripts. Unfortunately it was never renamed to reflect these expanded responsibilities, hence the confusion. Feel free to rename as you see fit!

Reply:
blech we need to reorganize!